libs/application/templates/rental-agreement/src/forms/rentalHousingInfo/rentalHousingCondition.ts

```diff
@@ -0,0 +1,79 @@
+import {
+  buildSubSection,
+  buildMultiField,
+  buildRadioField,
+  buildDescriptionField,
+  buildTextField,
+  buildFileUploadField,
+} from '@island.is/application/core'
+import * as m from '../../lib/messages'
+import build from 'next/dist/build'
+
+export const RentalHousingCondition = buildSubSection({
+  id: 'rentalHousingCondition',
+  title: m.housingCondition.subSectionName,
+  children: [
+    buildMultiField({
+      id: 'rentalHousingCondition',
+      title: m.housingCondition.pageTitle,
+      description: m.housingCondition.pageDescription,
+      children: [
+        buildDescriptionField({
+          id: 'rentalHousingConditionTitle',
+          title: m.housingCondition.inspectorTitle,
+          titleVariant: 'h3',
+          space: 1,
+        }),
+        buildRadioField({
+          id: 'rentalHousingConditionInspector',
+          title: '',
+          description: m.housingCondition.inspectorDescription,
+          options: [
+            {
+              value: 'contractParties',
+              label: m.housingCondition.inspectorOptionContractParties,
+            },
+            {
+              value: 'independentParty',
+              label: m.housingCondition.inspectorOptionIndependentParty,
+            },
+          ],
```

Comment:
Since this `buildSubSection` is slightly big, I would consider making a util function `getInspectorOptions`. Then this block would become:

`options: getInspectorOptions()`